.github/workflows/release.yml

```diff
@@ -95,6 +99,7 @@ jobs:
         name: nydus-artifacts
         path: nydus-static
     - name: upload artifacts
+      if: ${{ github.event_name == 'schedule' }}
```

Comment:
What is this PR trying to address?
Should the catching event is `push`?  Otherwise, the artifacts tarballs are overwritten every day?

Reply:
oops, you are right!